service/controller/resource/asgname/resource.go

```diff
@@ -83,7 +89,7 @@ func (r *Resource) ensure(ctx context.Context, obj interface{}) error {
 				{
 					Name: aws.String(fmt.Sprintf("tag:%s", key.TagStack)),
 					Values: []*string{
-						aws.String(key.StackTCNP),
```

Comment:
This was hardcoded to `tcnp`, but it's used by control plane drainer so I made it configurable. Before this, the drainer never found the control plane ASG name and therefore never drained the control plane nodes.